_community_projects/encrypted-repository-opensearch-plugin.md

```diff
@@ -0,0 +1,9 @@
+---
+name: Encrypted Repository Plugin for OpenSearch
+description: Encrypted Repository for OpenSearch is a plugin that offers client-side encryption for snapshotting repositories and could be combined with all official supported repository plugins: File Systsem, Google Cloud Storage, Amazon S3 and Microsoft Azure.
```

Comment:
when I build locally, it give this error:

"Error: YAML Exception reading /Users/freedak/Documents/OpenSearch/temp-willyborankin-add-encrypted-repository-opensearch-plugin-project-website/_community_projects/encrypted-repository-opensearch-plugin.md: (<unknown>): mapping values are not allowed in this context at line 3 column 195" 

also, shows as a blank tile locally on the "community_projects" page. Can you edit and push the PR once again?

Reply:
yes sorry will fix